srcpkgs/python3-pystray/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'python3-pystray'
+pkgname=python3-pystray
+version=0.17.4
+revision=1
+wrksrc="pystray-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-Pillow python3-pip python3-wheel"
+depends="python3 python3-six python3-Pillow python3-xlib python3-gobject
+libappindicator"
+short_desc="System tray icon library"
+maintainer="Neel Chotai <neel@chot.ai>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/moses-palmer/pystray"
+distfiles="https://github.com/moses-palmer/pystray/archive/v${version}.tar.gz"
+checksum=2929200adedffee710c5d08f854259c6365183ccdcb2784e7cc816add244f4a4
+make_check=extended
```

Comment:
Should be `no` with the a comment saying that it a) needs a tray and b) is interactive and expects user input